google/cloud/bigquery/v2/minimal/internal/common_v2_resources.h

```diff
@@ -155,7 +155,7 @@ struct ScriptOptions {
   std::chrono::milliseconds statement_timeout = std::chrono::milliseconds(0);
   std::int64_t statement_byte_budget = 0;
 
-  KeyResultStatementKind key_result_statement;
```

Comment:
What does the service do with `::UnSpecified()`?  Does it also fail like with an empty value?  If so, I would just not bother.  If it works then this is worthwhile.